apps/ledcontrol/uninstall

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+#Allow packages required by this app to be uninstalled
+sudo rm /usr/bin/ledcontrol
```

Comment:
Use `rm -f` to prevent this script from failing if `/usr/bin/ledcontrol` has already been deleted.

Reply:
✔